apollo-router/src/plugins/telemetry/snapshots/apollo_router__plugins__telemetry__otlp__metadata_map_serde__tests__serialize_metadata_map.snap

```diff
@@ -3,7 +3,8 @@ source: apollo-router/src/plugins/telemetry/otlp.rs
 expression: "std::str::from_utf8(&buffer).unwrap()"
 ---
 ---
-- foo: bar
-- foo: baz
-- bar: foo
-
+foo:
```

Comment:
If you want to keep ordering to not break the test then use `IndexMap` from https://docs.rs/indexmap/latest/indexmap/map/struct.IndexMap.html it should already exist in our deps